### PR TITLE
refactoring test sector

### PR DESCRIPTION
**Pull request by k4t4u**

### CODE REVIEW

File: src/test/java/pl/stormit/eduquiz/game/PlayGameIntegrationTest.java [hunk 1]
```diff
@@ -67,8 +67,7 @@ void shouldCreateGameCorrectly() {
         //when
```
Comment (RobertoJavaDev) [inline src/test/java/pl/stormit/eduquiz/game/PlayGameIntegrationTest.java:67]: Please correct each section: given/when/then ->
from:
//given
//when
//then

into:
// given
// when
// then
*Tomek pointed this out in code review

File: src/test/java/pl/stormit/eduquiz/quizcreator/controllers/AnswerApiControllerTest.java
Comment (RobertoJavaDev) [inline src/test/java/pl/stormit/eduquiz/quizcreator/controllers/AnswerApiControllerTest.java:1]: Delete redundant ";" in line 55.

File: src/test/java/pl/stormit/eduquiz/quizcreator/controllers/QuizApiControllerTest.java [hunk 2]
```diff
@@ -56,13 +59,10 @@ void shouldReturn200WhenFoundAllQuizzes() throws Exception {
...
         //when
-        ResultActions result = mockMvc.perform(get("/api/v1/quizzes")
-                .contentType(MediaType.APPLICATION_JSON));
+        ResultActions result = mockMvc.perform(get("/api/v1/quizzes").contentType(MediaType.APPLICATION_JSON));
 
         //then
-        result.andExpect(status().isOk())
-                .andExpect(content().string(containsString("PL")))
-                .andExpect(content().string(containsString("UK")));
+        result.andExpect(status().isOk()).andExpect(content().string(containsString("PL"))).andExpect(content().string(containsString("UK")));
```
Comment (RobertoJavaDev) [inline src/test/java/pl/stormit/eduquiz/quizcreator/controllers/QuizApiControllerTest.java:65]: I think the previous assert version is more readable:
result.andExpect(status().isOk())
                .andExpect(content().string(containsString("PL")))
                .andExpect(content().string(containsString("UK")));
               
You can see immediately what we are checking

File: src/test/java/pl/stormit/eduquiz/quizcreator/controllers/QuizApiControllerTest.java [hunk 3]
```diff
@@ -83,53 +82,38 @@ void shouldReturn200IfQuizIsFoundByIdCorrectly() throws Exception {
...
-                "Master", null, null, List.of(), List.of());
-
+        QuizRequestDto exemplaryQuizRequestDto = new QuizRequestDto("Master", null, null, List.of(), List.of());
         String dtoAsString = objectMapper.writeValueAsString(exemplaryQuizRequestDto);
 
         //when
-        MockHttpServletRequestBuilder content = post("/api/v1/quizzes")
-                .contentType(MediaType.APPLICATION_JSON)
-                .content(dtoAsString);
+        MockHttpServletRequestBuilder content = post("/api/v1/quizzes").contentType(MediaType.APPLICATION_JSON).content(dtoAsString);
```
Comment (RobertoJavaDev) [inline src/test/java/pl/stormit/eduquiz/quizcreator/controllers/QuizApiControllerTest.java:89]: The same like above

File: src/test/java/pl/stormit/eduquiz/quizcreator/controllers/QuizApiControllerTest.java [hunk 4]
```diff
@@ -83,53 +82,38 @@ void shouldReturn200IfQuizIsFoundByIdCorrectly() throws Exception {
...
-                "/api/v1/quizzes/{quizId}", FIRST_ID)
-                .contentType(MediaType.APPLICATION_JSON)
-                .content(objectMapper.writeValueAsString(exemplaryQuizRequestDto));
+        MockHttpServletRequestBuilder content = put("/api/v1/quizzes/{quizId}", FIRST_ID).contentType(MediaType.APPLICATION_JSON).content(objectMapper.writeValueAsString(exemplaryQuizRequestDto));
 
         //then
-        mockMvc.perform(content)
-                .andExpect(status().isOk())
-                .andDo(MockMvcResultHandlers.print());
+        mockMvc.perform(content).andExpect(status().isOk()).andDo(MockMvcResultHandlers.print());
```
Comment (RobertoJavaDev) [inline src/test/java/pl/stormit/eduquiz/quizcreator/controllers/QuizApiControllerTest.java:106]: The same like above

File: src/test/java/pl/stormit/eduquiz/quizcreator/controllers/UserApiControllerTest.java [hunk 5]
```diff
@@ -62,12 +65,7 @@ void shouldReturn200WhenFoundAllResults() throws Exception {
         MockHttpServletRequestBuilder content = get("/api/v1/users");
 
         //then
-        mockMvc.perform(content)
-                .andExpect(status().isOk())
-                .andExpect(content().string(containsString("Ananiasz")))
-                .andExpect(content().string(containsString("Wojski")))
-                .andExpect(content().string(containsString("Dajmiech")))
-                .andDo(MockMvcResultHandlers.print());
```
Comment (RobertoJavaDev) [inline src/test/java/pl/stormit/eduquiz/quizcreator/controllers/UserApiControllerTest.java:70]: The same like above

File: src/test/java/pl/stormit/eduquiz/quizcreator/controllers/UserApiControllerTest.java [hunk 6]
```diff
@@ -80,10 +78,7 @@ void shouldReturn200IfUserIsFoundByIdCorrectly() throws Exception {
         given(userService.getUser(FIRST_ID)).willReturn(expectedDtoUser);
 
         //then
-        mockMvc.perform(get(userUrl))
-                .andExpect(status().isOk())
-                .andExpect(content().string(containsString("Ananiasz")))
-                .andDo(MockMvcResultHandlers.print());
+        mockMvc.perform(get(userUrl)).andExpect(status().isOk()).andExpect(content().string(containsString("Ananiasz"))).andDo(MockMvcResultHandlers.print());
```
Comment (RobertoJavaDev) [inline src/test/java/pl/stormit/eduquiz/quizcreator/controllers/UserApiControllerTest.java:81]: The same like above

**Review by RobertoJavaDev — CHANGES_REQUESTED** [review]

**Review by RobertoJavaDev — APPROVED** [review]

**Review by Manes79 — APPROVED** [review]

**Review by JacekRG — APPROVED** [review]

**Pull request merged**